lib/components/viewers/RouteRow.js

```diff
@@ -16,9 +16,10 @@ import OperatorLogo from '../util/operator-logo'
 
 import RouteDetails from './route-details'
```

Comment:
Semantics: Could `RouteRow` use a `<summary>` element that contains a `<details>` section that includes `RouteDetails`?

Reply:
Is this the idea? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/summary

So instead of the expand that happens now, the button would be a summary dropdown?